tests/behat/features/theme.settings.feature

```diff
@@ -1,35 +1,51 @@
-      | header_logo_mobile.jpg  | public://civictheme_test/header_logo_mobile.jpg  | test_image.jpg |
-      | footer_logo_desktop.jpg | public://civictheme_test/footer_logo_desktop.jpg | test_image.jpg |
-      | footer_logo_mobile.jpg  | public://civictheme_test/footer_logo_mobile.jpg  | test_image.jpg |
+      | filename                | uri                                               | path           |
+      | header_logo_desktop.jpg  | public://civictheme_test/header_logo_desktop.jpg | test_image.jpg |
+      | header_logo_mobile.jpg  | public://civictheme_test/header_logo_mobile.jpg   | test_image.jpg |
+      | footer_logo_desktop.jpg | public://civictheme_test/footer_logo_desktop.jpg  | test_image.jpg |
+      | footer_logo_mobile.jpg  | public://civictheme_test/footer_logo_mobile.jpg   | test_image.jpg |
 
   @api @javascript
   Scenario: The CivicTheme theme settings has custom logo configuration
     Given I am logged in as a user with the "Site Administrator" role
     When I visit "/admin/appearance/settings/civictheme_demo"
-    Then I should see an "details#edit-logo" element
+    And I should see an "details#edit-logo" element
+    And I see field "Header desktop logo path"
     And I should see an "input[name='logo_path']" element
+    And should see an "input#edit-logo-path" element
+    And should not see an "input#edit-logo-path.required" element
+    And I see field "Header mobile logo path"
     And I should see an "input[name='civictheme_header_logo_mobile']" element
+    And should see an "input#edit-civictheme-header-logo-mobile" element
+    And should not see an "input#edit-civictheme-header-logo-mobile.required" element
+    And I see field "Footer desktop logo path"
     And I should see an "input[name='civictheme_footer_logo_desktop']" element
+    And should see an "input#edit-civictheme-footer-logo-desktop" element
+    And should not see an "input#edit-civictheme-footer-logo-desktop.required" element
+    And I see field "Footer mobile logo path"
     And I should see an "input[name='civictheme_footer_logo_mobile']" element
+    And should see an "input#edit-civictheme-footer-logo-mobile" element
+    And should not see an "input#edit-civictheme-footer-logo-mobile.required" element
+    And I see field "Logo alt attribute text"
     And I should see an "input[name='civictheme_site_logo_alt']" element
-
+    And should see an "input#edit-civictheme-site-logo-alt" element
+    And should not see an "input#edit-civictheme-site-logo-alt.required" element
+    And I see field "Footer background image path"
+    And I should see an "input[name='civictheme_site_logo_alt']" element
+    And should see an "input#edit-civictheme-footer-background-image" element
+    And should not see an "input#edit-civictheme-footer-background-image.required" element
+    When I fill in "Header desktop logo path" with "public://civictheme_test/header_logo_desktop.jpg"
```

Comment:
this test uses `public://` in the value of the asset path

we need to test that when a relative path is provided - it does not trigger an error - this was the main reason for this bug.

Could you please add another variant of this test where static assets would be used without a stream wrapper (`abc/def/file.png`).

Reply:
Added test for valid and failure static assets path